src/fsharp/FSharp.Compiler.Service.ProjectCracker.netcore/ProjectCrackerTool.fs

```diff
@@ -0,0 +1,361 @@
+﻿namespace Microsoft.FSharp.Compiler.SourceCodeServices.ProjectCrackerTool
```

Comment:
This duplication is intentional to minimize the impact on other projects, as it is a small refactoring (breaking the above mentioned Program.fs in two files so it can be reused), but yes, I can definitely do that, either here or in a subse